content/en/cd-as-a-service/concepts/deployment/traffic-management/smi-linkerd.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Traffic Management Using LinkerD
```

Comment:
It looks like the product documentation for Linkerd does not use a capital D so I would not use one.

Reply:
Good catch. I can't believe I made this mistake. Gaak!